emantomo/protocols/protocol_tomo_extraction_from_tomo.py

```diff
@@ -353,13 +354,14 @@ def readSetOfSubTomograms(self, tomoFile, outputSubTomogramsSet, coordSet, volId
                 ImageHandler.scaleSplines(subtomogram.getLocation()[1]+':mrc', fnSubtomo, dfactor)
                 subtomogram.setVolId(volId)
                 subtomogram.setLocation(fnSubtomo)
-            subtomogram.setCoordinate3D(coordSet[counter])
-            transformation = coordSet[counter]._eulerMatrix
+            subtomogram.setCoordinate3D(coordSet[idx])
```

Comment:
Could we just call coordSet[idx] once. This type of calls are, very easy to write but expensive to run. A set is no an "in memory collection". Every [idx] make a trip to sqlite to get the data and populate the object.

Reply:
Forget it, I was wrong.